package.json

```diff
@@ -27,7 +28,12 @@
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-register": "^6.18.0",
-    "eslint": "^3.12.2",
+    "eslines": "^0.0.12",
+    "eslint": "^3.13.0",
+    "eslint-eslines": "0.0.1",
```

Comment:
If we use `eslines` 0.0.13 and `eslint-eslines` 0.0.2 the warnings won't be shown. That's useful for CircleCI. It would also hide the warnings in pre-commit, though. I think that's OK and something that we can fix in a few days/weeks if we wanted a different behavior in pre-commit (I just didn't have the time to do it).